Source/Configuration/YPColors.swift

```diff
@@ -24,35 +27,35 @@ public struct YPColors {
     public var multipleItemsSelectedCircleColor: UIColor?
     
     /// The background color of the bottom of photo and video screens.
-    public var photoVideoScreenBackgroundColor: UIColor = .white
+    public var photoVideoScreenBackgroundColor: UIColor = .defaultBackgroundColor
```

Comment:
Color names must describe themself. Like "whiteBlue" or "brightOrange". Not defaultBackgroundColor.

Reply:
This guideline might be less strict now with dynamic colors, e.g. see naming conventions in [Apple's dynamic system colors](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/color/#dynamic-system-colors). Still, I updated to `.offWhiteOrBlack`.

Reply:
Indeed with dark mode we should be shifting to "semantic" names which describe the context rather than the color. Say "background" instead of "white".